### ISSUE TITLE
Operate cannot find Zeebe

### ISSUE THREAD

**Issue by jwulf** [zeebe-io/zeebe-helm#1]
I installed the helm charts following the docs on docs.zeebe.io, using `--name zeebe`. 

Afterwards, it all came up: 

```
➜ kubectl get svc
NAME                                  TYPE           CLUSTER-IP    EXTERNAL-IP     PORT(S)                                  AGE
elasticsearch-master                  ClusterIP      10.0.13.238   <none>          9200/TCP,9300/TCP                        132m
elasticsearch-master-headless         ClusterIP      None          <none>          9200/TCP,9300/TCP                        132m
zeebe                                 ClusterIP      None          <none>          9600/TCP,26500/TCP,26501/TCP,26502/TCP   132m
zeebe-nginx-ingress-controller        LoadBalancer   10.0.2.46     35.198.245.37   80:31634/TCP,443:30724/TCP               132m
zeebe-nginx-ingress-default-backend   ClusterIP      10.0.9.199    <none>          80/TCP                                   132m
zeebe-operate                         ClusterIP      10.0.3.41     <none>          80/TCP                                   132m
```

However, the Operate instance can't resolve the Zeebe broker.

@salaboy told me to do:

```
kubectl edit cm zeebe-operate
```

And change the double `zeebe-zeebe` to just `zeebe` to match the service.

I did this, then:

```
kubectl get pods
kubectl delete pod <operate-pod-id>
```

And it started working.

**Comment by salaboy** [comment]
working on this right now

**Comment by salaboy** [comment]
fixed in 0.0.17 release. 
You need to run `helm repo update` before installing again.. and probably remove your old release with` helm delete <release name> --no-hooks --purge` 

You will also need to delete the pvc as now stated in [Uninstalling Charts](https://github.com/zeebe-io/zeebe-helm#uninstalling-charts)
